lib/tasks/s3_file_conversion.rake

```diff
@@ -0,0 +1,39 @@
+require 'fileutils'
+include CarrierWave::MiniMagick
+
+namespace :s3 do
+  # Usage:
+  #     rake s3:file_conversion
+  desc "convert manually uploaded files to carrierwave objects"
+  task file_conversion: :environment do |task, args|
+    records = []
+
+    [Collection, Transcript].each do |klass|
+      klass.where.not(image_url: nil).find_each do |resource|
+        image_url = resource.read_attribute(:image_url)
```

Comment:
CarrierWave has its own image_url method that overrides this method too, so I have to use `read_attribute` :)